src/getting-started.md

```diff
@@ -0,0 +1,477 @@
+
+However, in practice, stage 1 should be sufficient. The first time you build,
+the stage-1 compiler will also be built.
+
+```sh
+# First build
+./x.py build --stage 1 src/tools/rustdoc
+
+# Subsequent builds
+./x.py build --stage 1 --keep-stage 1 src/tools/rustdoc
+```
+
+As with the compiler, you can do a fast check build:
+
+```sh
+./x.py check
+```
+
+Rustdoc has two types of tests: content tests and UI tests.
+
+```sh
+# Content tests
+./x.py test --stage 1 src/test/rustdoc
+
+# UI tests
+./x.py test --stage 1 src/test/rustdoc-ui
+
+# Both at once
+./x.py test --stage 1 src/test/rustdoc*
+```
+
+### Contributing code to other Rust projects
+
+There are a bunch of other projects that you can contribute to outside of the
+`rust-lang/rust` repo, including `clippy`, `miri`, `chalk`, and many others.
+
+These repos might have their own contributing guidelines and procedures. Many
+of them are owned by working groups (e.g. `chalk` is largely owned by
+WG-traits). For more info, see the documentation in those repos' READMEs.
+
+### Other ways to contribute
+
+There are a bunch of other ways you can contribute, especially if you don't
+feel comfortable jumping straight into the large `rust-lang/rust` codebase.
+
+The following tasks are doable without much background knowledge but are
+incredibly helpful:
+
+- [ICE-breakers Cleanup crew][iceb]: find minimal reproductions of ICEs, bisect
+  regressions, etc. This is a way of helping that saves a ton of time for
+  others to fix an error later.
+- Writing documentation: if you are feeling a bit more intrepid, you could try
+  to read a part of the code and write doc comments for it. This will help you
+  to learn some part of the compiler while also producing a useful artifact!
+- [Working groups][wg]: there are a bunch of working groups on a wide variety
+  of rust-related things.
+
+[iceb]: ./ice-breaker/cleanup-crew.md
+[wg]: https://rust-lang.github.io/compiler-team/working-groups/
+
+
+## Contributor Procedures
+
+There are some official procedures to know about. This is a tour of the
+highlights, but there are a lot more details, which we will link to below.
+
+### Code Review
+
+When you open a PR on the `rust-lang/rust` repo, a bot called `@rust-highfive` will
+automatically assign a reviewer to the PR. The reviewer is the person that will
+approve the PR to be tested and merged. If you want a specific reviewer (e.g. a
+team member you've been working with), you can specifically request them by
+writing `r? @user` (e.g. `r? @eddyb`) in either the original post or a followup
+comment.
+
+The reviewer may request some changes using the GitHub code review interface.
+They may also request special procedures (such as a [crater] run; [see
+below][break]) for some PRs.
+
+[break]: #breaking-changes
+
+When the PR is ready to be merged, the reviewer will issue a command to
+`@bors`, the CI bot. Usually, this is `@bors r+` or `@bors r=@user` to approve
```

Comment:
I think I've more often seen invocations ran as `@bors r=user` without the `@`, but I might be wrong and both syntaxes might be valid.
In case it's useful:
```suggestion
`@bors`, the CI bot. Usually, this is `@bors r+` or `@bors r=user` to approve
```